x/distribution/types/proposal.go

```diff
@@ -3,8 +3,38 @@ package types
 import (
 	"fmt"
 	"strings"
+
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
 )
 
+// GetTitle returns the title of a community pool spend proposal.
```

Comment:
this is something else right?

Reply:
https://github.com/cosmos/cosmos-sdk/pull/13262/files#r988413497